mongodb-setup/app/routes/api.create-demo-user.ts

```diff
@@ -0,0 +1,7 @@
+import { UserApi } from "~/user/api";
+export async function loader() {
```

Comment:
Keeping this consistent with the rest of the examples

```suggestion
export const loader = async () => {
```